.github/workflows/ci.yml

```
@@ -26,27 +26,120 @@ jobs:
+          ${{ runner.os }}-${{ runner.arch }}-go-
+        enableCrossOsArchive: false
     
-    - name: Download dependencies
-      run: go mod download
+    - name: Download dependencies (Unix)
+      if: runner.os != 'Windows'
+      run: |
+        timeout_duration=300
+        max_retries=3
+        retry_count=0
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)